packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -445,10 +445,22 @@ class CodePushClient {
   }
 
   /// List all release for the provided [appId].
-  Future<List<Release>> getReleases({required String appId}) async {
-    final response = await _httpClient.get(
-      Uri.parse('$_v1/apps/$appId/releases'),
+  Future<List<Release>> getReleases({
+    required String appId,
+    bool sideloadableOnly = false,
+  }) async {
+    var uri = Uri.parse(
+      '$_v1/apps/$appId/releases',
     );
+    if (sideloadableOnly) {
+      uri = uri.replace(
+        queryParameters: {
+          if (sideloadableOnly) 'sideloadable': 'true',
```

Comment:
```suggestion
          'sideloadable': 'true',
```
Do we need another check?

Reply:
Good call, fixed.